float32 x_pos
float32 y_pos
float32 theta
---
bool reached 
---
float32 x_final_pose
float32 y_final_pose
float32 theta_final_pose
